app/src/main/res/layout/rc_phase_shift_setup.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="5dp">
+
```

Comment:
I will reuse it. Since this layout is being used in a lot of experiments, I think we should rename it instead of using `diode_setup`.